.github/workflows/test-partner-driver.yml

```diff
@@ -0,0 +1,50 @@
+name: Test All Partner Drivers
+
+on:
+  schedule:
+    - cron: "0 * * * *"
+  push:
+
+jobs:
+  setup:
+    runs-on: ubuntu-latest
+    outputs:
+      versions: ${{ steps.versions.outputs.value }}
+      drivers: ${{ steps.drivers.outputs.value }}
+    steps:
```

Comment:
little tip: you can set this once for the whole job at the job level, which also makes it easier to read:

```
env:
  CURRENT_VERSION: ${{ vars.CURRENT_VERSION }}

Reply:
also, we should check and make sure this variable is an org-level variable, and not specific to each repo so we don't have to keep them in sync, which we will 100% fail to do at some point.

Reply:
It is org level.